provisioning/tasks/config.yml

```diff
@@ -12,5 +12,5 @@
   with_fileglob:
     - "{{ config_dir }}/config.yml"
     - "{{ config_dir }}/local.config.yml"
-    - "{{ config_dir }}/secrets.yml"
+    - "{{ config_dir }}/{{ lookup('env', 'DRUPALVM_ENV')|default(drupalvm_env, true)|default(ansible_env.DRUPALVM_ENV)|default(omit) }}.secrets.yml"
```

Comment:
Could you keep the old behavior as well as we talked about in https://github.com/geerlingguy/drupal-vm/issues/1522#issuecomment-333314535

Reply:
True, that sounds like a good solution but I do see a use case for separate `<env>.secrets.yml` files, eg separate staging/prod environments. But maybe we shouldn't add more features unless someone actually asks for it.

Reply:
It would be nice to have... but if it makes things harder, let's keep the current level of support only.